app/controllers/pages_controller.rb

```diff
@@ -36,18 +36,24 @@ def i_belong
       posters_link_method = :get
       cta_link_path = i_belong_path
       cta_link_method = :get
+      handbook_download_url = 'https://static.teachcomputing.org/I+Belong+Handbook.pdf'
```

Comment:
Can you use the helper `helpers.i_belong_handbook_url` (defined in S3LinkHelper) rather than specifying the URL again here.

Reply:
resolved thank you